p11-kit/list-objects.c

```diff
@@ -170,9 +236,10 @@ print_object (p11_list_printer *printer,
 	CK_MECHANISM_TYPE mechanism_type;
 	CK_BBOOL trusted, local, token, private, modifiable, copyable, destroyable;
 	CK_DATE start_date, end_date;
-	char label[128] = { '\0' };
-	char application[128] = { '\0' };
-	char id[128] = { '\0' };
+	char label[BUFFER_SIZE] = { '\0' };
```

Comment:
Not originated in this PR, but shouldn't we allow those attributes of arbitrary length?

Reply:
We probably should, yes.

Reply:
done